api/nntrainer-api-common.h

```diff
@@ -239,6 +239,16 @@ typedef enum {
          required for inference and training. */
 } ml_train_model_format_e;
 
+/**
+ * @brief Defining Macros for Exporting Dynamic Library in Windows.
+ * @note Macro is meaningful only if platform is windows.
+ */
+#ifdef _WIN32
+#define ML_API __declspec(dllexport)
```

Comment:
let's check this declaration after paper work is done.
nntrainer-api-common.h is header for Tizen native API set, so I think this may raise an error when checking Tizen release process.

Recommend: add this declaration in nntrainer internal header file, not in Tizen header.

Reply:
Changed the position of the define to api/ccapi/include/common.h.